app/models/tag.rb

```diff
@@ -190,9 +192,7 @@ def check_synonym
     if !self.new_record? && self.name_changed?
       # ordinary wranglers can change case and accents but not punctuation or the actual letters in the name
       # admins can change tags with no restriction
-      unless User.current_user.is_a?(Admin) || (self.name.downcase == self.name_was.downcase) || (self.name.mb_chars.normalize(:kd).gsub(/[^\x00-\x7F]/u,'').downcase.to_s == self.name_was.mb_chars.normalize(:kd).gsub(/[^\x00-\x7F]/u,'').downcase.to_s)
-        self.errors.add(:name, "can only be changed by an admin.")
-      end
+      self.errors.add(:name, "can only be changed by an admin.") unless User.current_user.is_a?(Admin) || (self.name.downcase == self.name_was.downcase) || (UnicodeUtils.casefold(self.name).mb_chars.normalize(:kd).gsub(/[\u0300-\u036F]/u, "").to_s == UnicodeUtils.casefold(self.name_was).mb_chars.normalize(:kd).gsub(/[\u0300-\u036F]/u, "").to_s)
```

Comment:
For readability, would it be worthwhile to move the normalization process into its own method?

`(UnicodeUtils.casefold(self.name).mb_chars.normalize(:kd).gsub(/[\u0300-\u036F]/u, "").to_s` is a lot, especially twice in a row. 

@zz9pzza also wanted me to pass along a question: is the `to_s` still necessary?

Reply:
I've refactored it out into a method `normalize_for_tag_comparison`. I agree that's a lot more readable.

zz9pzza is also totally right: the `to_s` was redundant, so I've removed that.

Thank you!